pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -85,9 +90,13 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "https://github.com/bazelbuild/bazel/releases/download/${version}/${name}-dist.zip";
-    sha256 = "0hannnvia8rvmi2v5d97j1f6wv0m1kxkd5hq4aqp0dqjr0ka4q38";
+    sha256 = "11gsc00ghxqkbci8nrflkwq1lcvqawlgkaryj458b24si6bjl7b2";
   };
 
+  # Necessary for the tests to pass on Darwin with sandbox enabled.
+  # Bazel starts a local server and needs to bind a local address.
+  __darwinAllowLocalNetworking = true;
```

Comment:
I assume this is inherent with bazel, is this a random or fixed port? If it's static hydra and local builds will have problems if multiple bazel builds are running at the same time.

Reply:
Yes, it is inherent to Bazel. It uses a daemon for faster builds. There used to be a `--batch` option, but it is being removed: https://docs.bazel.build/versions/master/command-line-reference.html#flag--batch

The code that I looked at seems to allow the kernel to choose a random port.